hooks/pairs/newAmbient/useAmbientPools.ts

```diff
@@ -40,7 +40,7 @@ export default function useAmbientPools(
           params.userEthAddress
         );
         if (rewardsQuery.error) throw rewardsQuery.error;
```

Comment:
does this cause the error to still be thrown? 

in case of error, it should just show 0